WISE-Community/WISE
# Make changes to WISE5 side for the WISE4 phaseout

Add this property to application.properties
wise4.hostname=http://localhost:8080/legacy

Test these in the WISE5 site

Students
 - Make sure they can't add a WISE4 run
 - Make sure launching a WISE4 run launches using the legacy page

Teachers
 - In the library, make sure they can't start a WISE4 run
 - In the library, make sure a WISE4 project details does not show the dropdown for Copy/Edit/Share
 - In the run list, make sure the drop down for a WISE4 run only shows Preview/Unit Info/Report Problem
 - In the run list, make sure launching a WISE4 run Classroom Monitor launches using the legacy page
 - In the run list, WISE4 runs do not have a link to manage students where it shows "x students"


Closes #2243
